05-http-server-streams/01-file-server-get/server.js

```diff
@@ -1,22 +1,52 @@
 const url = require('url');
 const http = require('http');
 const path = require('path');
+const fs = require('fs');
 
 const server = new http.Server();
 
+const sendFile = (res, filepath) => {
+  const readStream = fs.createReadStream(filepath);
+
+  readStream.on('open', () => {
+    readStream.pipe(res);
+  });
+
+  readStream.on('error', (err) => {
+    res.statusCode = 500;
+    res.end(err);
```

Comment:
Should I pass the server error to the client?

Reply:
The task doesn't require it, however, usually, it makes sense.
The only note - do not pass the error as is. 
From security and DX (developer experience), it will be better to send to a client message that comprehensive enough for a client, but do not expose internal state/information/etc (like stack trace)